spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/MigrateTableProcedure.java

```diff
@@ -91,9 +92,13 @@ public InternalRow[] call(InternalRow args) {
     }
 
     boolean dropBackup = args.isNullAt(2) ? false : args.getBoolean(2);
+    String backupSuffix = args.isNullAt(3) ? null : args.getString(3);
 
     MigrateTableSparkAction migrateTableSparkAction =
         SparkActions.get().migrateTable(tableName).tableProperties(properties);
+    if (backupSuffix != null) {
+      migrateTableSparkAction = migrateTableSparkAction.withBackupSuffix(backupSuffix);
+    }
 
     MigrateTable.Result result;
```

Comment:
Can you also change this to the following to match the style above

if (dropBackup) {
migrateTableSparkAction = migrateTableSparkAction.dropBackup()
}

MigrateTable.Result result = migrateTableSparkAction.execute();